tests/queries/0_stateless/02232_non_negative_derivative.sql

```diff
@@ -0,0 +1,39 @@
+DROP TABLE IF EXISTS nnd;
```

Comment:
Perhaps add also tests which check for the other conditions, e.g. what happens if one/two/four arguments are provided or if some argument datatypes are not as expected etc.